studio/pages/projects.tsx

```diff
@@ -38,7 +38,7 @@ const ProjectsPage: NextPageWithLayout = () => {
 
 ProjectsPage.getLayout = (page) => (
   <AccountLayout
-    title="Supabase"
+    title="Projects"
```

Comment:
Because the "Projects" page is the home page for the dashboard, maybe this should say "Dashboard" instead of "Projects".

cc @MindTomato

Reply:
Thanks @alaister Should I change `Projects` in `breadcrumbs` to `dashboard` too?